packages/x-date-pickers/src/locales/jaJP.ts

```diff
@@ -0,0 +1,61 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { CalendarPickerView } from '../internals/models';
+
+const jaJPPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: '先月',
+  nextMonth: '来月',
+
+  // View navigation
+  openPreviousView: '前の画面を開く',
+  openNextView: '次の画面を開く',
+  calendarViewSwitchingButtonAriaLabel: (view: CalendarPickerView) =>
+    view === 'year'
+      ? '年選択画面からカレンダー画面に切り替える'
+      : 'カレンダー画面から年選択画面に切り替える',
+  // inputModeToggleButtonAriaLabel: (isKeyboardInputOpen: boolean, viewType: 'calendar' | 'clock') => isKeyboardInputOpen ? `text input view is open, go to ${viewType} view` : `${viewType} view is open, go to text input view`,
```

Comment:
Thank you for you contribution! 🙏 
Could you by any chance also add translations to the commented out lines?
Or do you not know how to translate those? 🤔

Reply:
Thanks for review!
Added missing translations 🙏